src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -109,6 +110,21 @@ public override TypeSyntax VisitDynamicType(IDynamicTypeSymbol symbol)
                     SyntaxFactory.IdentifierName("dynamic"), symbol);
             }
 
+#if !CODE_STYLE
+
+            public override TypeSyntax VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
```

Comment:
❔ Is this code covered by the generate method test above?

Reply:
Yes, although I think we also have direct tests I should update too...

Reply:
Actually I guess we don't -- we just have a huge pile of tests that call these helpers indirectly.